.circleci/config.yml

```diff
@@ -101,6 +101,9 @@ jobs:
     <<: *container_config_node8
     steps:
       - *attach_workspace
+      - add_ssh_keys:
+          fingerprints:
+            - "2b:98:17:21:34:bf:5d:3b:15:a5:82:77:90:11:03:e9"
```

Comment:
where's this key stored? in the circle ui?

Reply:
Yeah, there is a separate SSH key settings page, out of reach of Vault (@sjparkinson - any better ways of doing this?)